checks/evaluation/pinned_dependencies.go

```diff
@@ -121,14 +121,21 @@ func PinningDependencies(name string, c *checker.CheckRequest,
 		return checker.CreateRuntimeErrorResult(name, err)
 	}
 
+	// Pip installs.
+	pipScore, err := createReturnForIsPipInstallPinned(pr, dl)
+	if err != nil {
```

Comment:
Thanks! Can you also please include tests for this?

Reply:
Yes, sure! Thanks for reviewing Naveen!

Reply:
@naveensrinivasan I added some test cases! I don't know if it's possible to test the case where a dependency ends in "Inconclusive score: -1" or in error. Let me know what you think.

Reply:
Sounds good. Thanks